sui/its-example.js

```diff
@@ -130,7 +138,7 @@ async function sendToken(keypair, client, contracts, args, options) {
             TokenId,
             Coin,
             destinationChain,
-            destinationAddress,
+            destinationTokenAddress,
```

Comment:
this is not a token address, it's the recipient, so `encodeITSDestination` should be used